applications/surgical_scene_recon/run_complete_workflow.py

```
@@ -0,0 +1,403 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.14.5)

The file is executable but no shebang is present

(EXE002)